Zend/zend_inheritance.c

```diff
@@ -1154,8 +1154,8 @@ static void zend_add_trait_method(zend_class_entry *ce, const char *name, zend_s
 			/* Make sure the abstract declaration is compatible with previous declaration */
```

Comment:
Due to this else if, are we not going to miss a check in a case like this?

```php
abstract class A {
    abstract function foo($x);
}
trait T {
    abstract function foo($x = 0);
}
abstract class C extends A {
    use T;
}
```

Assuming I'm understanding the desired behavior correct (that is, abstract methods in traits impose an additional constraint, they do not simply override), this should fail, right?